sequins.go

```diff
@@ -56,7 +55,16 @@ func (s *sequins) start(address string) error {
 	s.started = now
 	s.updated = now
 
-	defer s.index.Close()
+	return nil
+}
+
+func (s *sequins) start(address string) error {
+	// TODO: we may need a more graceful way of shutting down, since this will
```

Comment:
Yeah, that's more or less what I'm alluding to. I just didn't want to stuff this into the same PR.